mdoc/Mono.Documentation/Frameworks/FrameworkTypeEntry.cs

```diff
@@ -8,6 +8,10 @@ namespace Mono.Documentation
 	class FrameworkTypeEntry : IComparable<FrameworkTypeEntry>
 	{
 		SortedSet<string> members = new SortedSet<string> ();
+		SortedSet<string> memberscsharpsig = new SortedSet<string> ();
+
+		CSharpFullMemberFormatter formatter = new CSharpFullMemberFormatter ();
```

Comment:
good point ... there wasn't really any rhyme or reason to it, other than the fact that the C# sig is used in several other spots for bookkeeping purposes and I just defaulted to it. But you're right, `ILSig` is probably a better unit to use going forward :)